### ISSUE TITLE
Change copy of downloads page

### ISSUE THREAD

**Issue by davidlago** [ComputeForHumanity/compute-for-humanity-app#23]
This is totally subjective, so feel free to ignore. In the downloads page, if the website didn't detect your OS correctly, the button to switch says "Sorry, I don't have a Mac". For some reason it sounds a bit like the user is saying she is sorry for not having a Mac, as if theirs is a lesser option :)

![screen shot 2015-10-22 at 9 06 47 am](https://cloud.githubusercontent.com/assets/5722719/10665924/72d88f42-789c-11e5-83dd-3c448394284d.png)

**Comment by JacobEvelyn** [comment]
Yeah I thought about that when I wrote it initially but I wasn't sure how to make it friendly ("I don't have a Mac" seems cold on its own).

And there actually isn't any OS detection going on, but interesting thought!
